src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_functionapp_commands.py

```diff
@@ -526,6 +527,28 @@ def test_functionapp_windows_runtime(self, resource_group, storage_account):
         self.cmd('functionapp config appsettings list -g {} -n {}'.format(resource_group, functionapp_name), checks=[
             JMESPathCheck("[?name=='FUNCTIONS_WORKER_RUNTIME'].value|[0]", 'node')])
 
+    @ResourceGroupPreparer(location=FUNCTIONAPP_EUAP_LOCATION)
+    @StorageAccountPreparer()
+    def test_functionapp_create_with_appcontainer_managed_environment(self, resource_group, storage_account):
```

Comment:
Can you run `azdev test test_functionapp_create_with_appcontainer_managed_environment --discover` and check in the yaml file that gets generated in the tests/latest/recordings folder?